system/maintenance-controller/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: maintenance-controller
+description: A controller to manage node maintenance
+type: application
+version: 0.1.0
+appVersion: "0.1"
+home: https://github.com/sapcc/maintenance-controller
+dependencies:
+  - name: flatcar-linux-update-agent
```

Comment:
The flatcar-linux-update-agent is not a hard dependency of the maintenance controller. So I suggest to make it at optional by adding `condition: flatcar-linux-update-agent.enabled` and having the `enabled` value default to `true` in the charts values.yaml.